src/PowerShellEditorServices/Services/PowerShellContext/Handlers/GetVersionHandler.cs

```diff
@@ -118,7 +118,7 @@ private async Task CheckPackageManagement()
                 {
                     StringBuilder errors = new StringBuilder();
                     await _powerShellContextService.ExecuteScriptStringAsync(
-                        "powershell.exe -NoLogo -NoProfile -Command 'Install-Module -Name PackageManagement -Force -MinimumVersion 1.4.6 -Scope CurrentUser -AllowClobber'",
+                        "powershell.exe -NoLogo -NoProfile -Command '[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; Install-Module -Name PackageManagement -Force -MinimumVersion 1.4.6 -Scope CurrentUser -AllowClobber' -Repository PSGallery",
```

Comment:
This command isn't quite right. You have the `'` before the `-Repository`

Reply:
> This command isn't quite right. You have the `'` before the `-Repository`

Sorry, my mistake : ( will correct asap